### ISSUE TITLE
Button on the top toolbar to start playing last played file

### ISSUE THREAD

**Issue by xCipheredz** [flackbash/AudioAnchor#104]
It's sort of a hassle to seek out the last played file, there is a feature in VLC where there's a button to start playing the last played file on the top toolbar. Though this exists for AudioAnchor in the form of the red button on the bottom right, it goes away when you close the app. Thanks!

**Comment by flackbash** [comment]
Yes, the red button is only to control the currently active audio file. The audio file should be active as long as the notification is shown. (Not in Android 11 though, things are weird there)

I get your point and will consider this for future improvements.
